default-menu-auto-hide-fedora.ks.in

```diff
@@ -0,0 +1,17 @@
+#version=DEVEL
+#test name: default-menu-auto-hide
+%ksappend repos/default.ks
+
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+%post --nochroot
```

Comment:
I am not sure why you run the script outside of the chroot. It doesn't seem to be necessary.

Reply:
Fixed.